File: GPT/Uebung12/NatSum.java
public class NatSum {
  public static void main(String[] args) {
    int sum = 0;
    String ret = "";
    for (int i = 1; i < Integer.parseInt(args[0]) + 1; i++) {
      if (ret != "") {
        ret += " + " + String.valueOf(i);
      } else {
        ret += String.valueOf(i);
      }
      sum += i;
    }

    ret += " = " + String.valueOf(sum);

    System.out.println(ret);
  }
}

File: GPT/Uebung12/NatSum.py
import sys

sum = 0
ret = ""
for i in range(1, int(sys.argv[1]) + 1):
    if ret != "":
        ret += " + " + str(i)
    else:
        ret += str(i)
    sum += i

ret += " = " + str(sum)

print(ret)